reana_client/cli/workflow.py

```diff
@@ -143,8 +153,13 @@ def workflow_workflows(
     for p in ctx.params:
         logging.debug("{param}: {value}".format(param=p, value=ctx.params[p]))
     type = "interactive" if sessions else "batch"
-    if _filter:
-        parsed_filters = parse_parameters(_filter)
+
+    status_filter = None
+    search_filter = None
+    if filters:
+        status_filter, search_filter = parse_filter_parameters(filters)
+    if format:
+        parsed_formatting_filters = parse_parameters(format)
```

Comment:
What do you think about renaming this function to `parse_format_parameters`? So we have both `parse_filter_parameters` and `parse_format_parameters`, each one corresponding to a different option logic.

Reply:
Done 👍